src/metadata.rs

```diff
@@ -121,6 +121,8 @@ pub struct AttackMeta {
     pub hitbox: ColliderMeta,
     pub hitstun_duration: f32,
     pub velocity: Option<Vec2>,
+    pub spritesheet: Option<FighterSpritesheetMeta>,
+    pub projectile_frames: Option<AttackFrames>,
```

Comment:
Could you refactor this this to remove the optional spritesheet from `AttackMeta` and move that to an item definition?

Reply:
For sure, just not sure how would the boss get the item, a field on the fighter? or maybe on the attack?

Reply:
You dont have to have it pick up or hold the item or anything, just spawn it when the boss attacks